static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -205,3 +219,14 @@ const StyledButton = styled(Button)`
     linear-gradient(to right, ${p => p.theme.background}, ${p => p.theme.pink400}20);
   color: ${p => p.theme.pink400};
 `;
+
+const HeaderContainer = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
```suggestion
  gap: ${space(1)};
```
Why space between here? Would look more consistent with the badge next to the title